modules/@apostrophecms/asset/index.js

```diff
@@ -239,6 +244,58 @@ ${bundle}
       },
       scripts: function (when) {
         return self.scriptsHelper(when);
+      },
+      reload() {
+        return self.apos.template.safe(`
+<script>
+  (function() {
+    var reloadId;
+    var fast = '1';
```

Comment:
If `fast` is simply for the initial run, it would be clearer to name that as such.

Reply:
`fast` also gets used after any error, to keep an eye out for the process coming back.